support/doc_config_schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "title": ".doc_config.yaml Validation Schema",
+  "description": "This schema describes the shape of our internal doc configs. It is used to validate configs so that the build script doesn't need to do this checking.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "unevaluatedProperties": false,
```

Comment:
I see that `unevaluatedProperties` appears here, but `additionalProperties` is also set. I only just now learned that `unevaluatedProperties` existed but was left with the impression that having both set to `false` would be redundant. Am I missing a critical element to `unevaluatedProperties`'s behavior? For example, maybe it's recursive?

Reply:
Ah, sorry about that. I should omit `unevaluatedProperties` -- my understanding accords with yours. I was reading about it today and earlier I was a little bit confused about what it did (I thought it might impact metaschema validation, which it does not).

As I understand it, it's only useful when you want to capture the suite of things which didn't validate against `additionalProperties`. e.g. a schema containing `"additionalProperties": {"type": "string"}` may leave `unevaluatedProperties` which can be passed along as part of the output of a schema validator.

I'll remove it though. :+1: